app/models/rental.rb

```diff
@@ -0,0 +1,34 @@
+class Rental < ApplicationRecord
+  belongs_to :customer
+  belongs_to :video
+
+  def self.inventory_check_out(rental) 
+    customer = Customer.find_by(id: rental.customer_id)
+    video = Video.find_by(id: rental.video_id)
+
+    if video.available_inventory > 0 && customer && video
+       customer.videos_checked_out_count += 1 
+       customer.save
+       video.available_inventory -= 1
+       video.save
+      return true 
+    else
+      return false 
```

Comment:
I really like returning a boolean value to indicate success or failure here though.